arelle/Validate.py

```diff
@@ -566,6 +566,7 @@ def determineTestStatus(self, modelTestcaseVariation, errors):
         elif isinstance(expected,(QName,_STR_BASE,dict,list)): # string or assertion id counts dict
```

Comment:
Random Line:

Better check that the tests and data don't get packaged with the releases

Reply:
They most certainly do. Will have to fix that once we get to the building point.